### ISSUE TITLE
allow properties to be overriden from command line

### ISSUE THREAD

**Issue by jl-DaDar** [awslabs/amazon-kinesis-client-python#69]
I want to run multiple KCL consumers in docker. In order to have multple KCL consumer client, I need each of them using different application name with same stream. However, I am unable to set application name from command line, and that means I will need to have multiple properties file, one for each consumer to have different application name. 

This is a problem for me because I will then have to know how many consumer I need to run before hand, or I will need to rebuild docker image to have more properties file with different application name. Obviously there are ways to get around these problem, but they are a pain in the ass. The best way is to allow overriding of properties attribute from command line. Thanks.

**Comment by jl-DaDar** [comment]
i tried -DapplicationName=<appname>     it does not work

**Comment by pfifer** [comment]
I don't believe this is supported currently, but it is something we can look at adding support for in the future.

**Comment by ravi0912** [comment]
Do we have support for this, I have the same requirements.
